net-libs/meanwhile/meanwhile-1.1.1.ebuild

```diff
@@ -31,12 +31,18 @@ PATCHES=(
 
 src_prepare() {
 	default
+	# bug #935514
+	sed -i -e "s|-g -O0 -Weverything||g" configure.ac || die
 	eautoreconf
 }
 
 src_configure() {
 	append-cflags -fno-tree-vrp
 
+	if use debug; then
+		append-cflags -DDEBUG;
+	fi
+
 	econf \
 		--enable-doxygen=$(usex doc) \
 		$(use_enable debug)
```

Comment:
The -DDEBUG isn't needed if you pass `$(use_enable debug)`.

Reply:
oh.. i see, fixed it now.